package.json

```diff
@@ -27,12 +27,12 @@
     "copy-files": "copyfiles -u 2 ./src/prisma/* ./dist/prisma/ && copyfiles -u 3 ./src/prisma/migrations/**/*.sql ./dist/prisma/migrations/"
   },
   "dependencies": {
-    "@aws-sdk/client-kms": "^3.398.0",
+    "@aws-sdk/client-kms": "^3.679.0",
     "@bull-board/fastify": "^5.21.1",
     "@cloud-cryptographic-wallet/cloud-kms-signer": "^0.1.2",
     "@cloud-cryptographic-wallet/signer": "^0.0.5",
     "@fastify/basic-auth": "^5.1.1",
-    "@fastify/cookie": "^8.3.0",
+    "@fastify/cookie": "^9.2.0",
```

Comment:
This is the only major version bump and impacts thirdweb-dev/auth:
`=> Found "@thirdweb-dev/auth#@fastify/cookie@9.3.1"`

I tested dashboard auth and access token auth to confirm there were no breaking changes.